.evergreen/run-tests.sh

```diff
@@ -134,6 +134,10 @@ if [ -n "$TEST_ENCRYPTION" ]; then
     python -c "import pymongocrypt; print('libmongocrypt version: '+pymongocrypt.libmongocrypt_version())"
     # PATH is updated by PREPARE_SHELL for access to mongocryptd.
 
+    # Get access to the AWS temporary credentials:
+    # CSFLE_AWS_TEMP_ACCESS_KEY_ID, CSFLE_AWS_TEMP_SECRET_ACCESS_KEY, CSFLE_AWS_TEMP_SESSION_TOKEN
+    python -m pip install boto3
```

Comment:
Should we add this to `test-encryption-requirements.txt` instead?

Reply:
Done.